Assets/Scripts/BlackHolePull.cs

```diff
@@ -32,8 +35,9 @@ private void FixedUpdate()
             var singularityPosition = transform.position;
             var pulledBodyPosition = pulledBody.transform.position;
             var pullDirection = (singularityPosition - pulledBodyPosition).normalized;
-            // TODO: optimize the pull force equation
-            pulledBody.AddForce(Time.fixedDeltaTime * gravitationalForce * pullDirection, ForceMode.Acceleration);
+            var pullDistance = Vector3.Distance(pulledBodyPosition, blackHoleCenter.ClosestPoint(pulledBodyPosition));
+            var pullForce = Time.fixedDeltaTime * gravitationalForce * GravForceMultiplier / Mathf.Pow(pullDistance, 2);
+            if (!float.IsNaN(pullForce) && !float.IsInfinity(pullForce)) pulledBody.AddForce(pullForce * pullDirection, ForceMode.Acceleration);
```

Comment:
Do you maybe want to factor in mass of pulled bodies by using the normal ForceMode? Asteroids probably shouldn't be pulled as hard as the player will...?

Reply:
I think this effect is fine gameplay-wise, at lease for now. It would require a lot of fine tuning if we factored in mass because if we're being realistic the player would get pulled super fast and the asteroids would barely be pulled at all, which isn't very fun...

Reply:
I put a ticket in the backlog: https://chilipot.atlassian.net/browse/PROP-37